wom/src/main/scala/wdl4s/wdl/AstTools.scala

```diff
@@ -306,7 +306,7 @@ object AstTools {
         && a.getAttribute("rhs").isTerminal => a.getAttribute("rhs").asInstanceOf[Terminal]
     }
 
-    def fullVariableReferenceString = terminal.getSourceString + terminalSubIdentifier.map("." + _.getSourceString).getOrElse("")
+    def fullVariableReferenceString = terminal.getSourceString + terminalSubIdentifier.fold(""){ "." + _.getSourceString }
```

Comment:
I don't personally all that much, like i said I like the idea of `fold`s like this a lot on paper, I just aesthetically prefer the other. Either way is fine by me though